README.md

```diff
@@ -1,14 +1,14 @@
 # Google Drive support for Jupyter Notebook
 
-This repository contains a custom
+This repository contains customs
```

Comment:
Errr, looks like "a custom" is right.

"...contains a custom Contents class that..."